tidy3d/components/data/data_array.py

```diff
@@ -700,6 +700,24 @@ class FieldProjectionAngleDataArray(DataArray):
     _data_attrs = {"long_name": "radiation vectors"}
 
 
+class DirectivityDataArray(DataArray):
+    """Directivity in frequency domain as a function of angles theta and phi.
```

Comment:
What are the units?

Reply:
Directivity is a dimensionless quantity defined as the ratio of the radiation intensity in a given direction to the average radiation intensity over all directions.

Reply:
Sounds good, could you copy that sentence into the docstring? will be helpful in case it is not clear for someone.